### PR TITLE
Enhancements and simplifications

### PR DESCRIPTION
**Pull request by MarkRivers**
Added RELEASE.md that describes the changes in this PR.

### CODE REVIEW

File: iocBoot/iocLabJack/generate_T7_2_and_3_substitutions.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cp LabJack_T7_1.substitutions LabJack_T7_2.substitutions
+cp LabJack_T7_1.substitutions LabJack_T7_3.substitutions
+sed -i s/:1:/:2:/ LabJack_T7_2.substitutions
+sed -i s/_1_/_2_/ LabJack_T7_2.substitutions
+sed -i s/:1:/:3:/ LabJack_T7_3.substitutions
+sed -i s/_1_/_4_/ LabJack_T7_3.substitutions
```
Comment (keenanlang) [inline iocBoot/iocLabJack/generate_T7_2_and_3_substitutions.sh]: Wouldn't this be s/\_1\_/\_3\_/ ?

File: iocBoot/iocLabJack/st.cmd.linux [hunk 2]
```diff
@@ -16,15 +17,19 @@ LabJackApp_registerRecordDeviceDriver(pdbbase)
...
+
+epicsEnvSet("N", "1")
+epicsEnvSet("IP_ADDRESS", "gse-labjack1")
+< LabJack_T7.cmd
+epicsEnvSet("N", "2")
+epicsEnvSet("IP_ADDRESS", "gse-labjack2")
+< LabJack_T7.cmd
+#epicsEnvSet("N", "3")
+#epicsEnvSet("IP_ADDRESS", "gse-labjack3")
+#< LabJack_T7.cmd
```
Comment (keenanlang) [inline iocBoot/iocLabJack/st.cmd.linux]: Maybe switch these to iocshLoad?

iocshLoad("LabJack_T7.com", "N=1, IP_ADDRESS=gse-labjack1")
iocshLoad("LabJack_T7.com", "N=2, IP_ADDRESS=gse-labjack2")
\#iocshLoad("LabJack_T7.com", "N=3, IP_ADDRESS=gse-labjack3")

File: iocBoot/iocLabJack/generate_T7_2_and_3_substitutions.sh (hunk 1 above)
Comment (MarkRivers) [reply iocBoot/iocLabJack/generate_T7_2_and_3_substitutions.sh]: Thanks for catching that.  I had only tested LabJack_T7_2.substitutions, not 3.

File: iocBoot/iocLabJack/st.cmd.linux (hunk 2 above)
Comment (MarkRivers) [reply iocBoot/iocLabJack/st.cmd.linux]: Thanks for the suggestion, I have made that change.

**Comment by MarkRivers** [comment]
I believe this is ready to be merged.

**Pull request merged**